<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerMainWidget</class>
 <widget class="QWidget" name="PlayerMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="MiniPlayerScreenWidget" name="ScreenWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="buttonWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>560</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <widget class="QSlider" name="playProgressSlider">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>550</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::add-page:Horizontal{
background-color: rgb(87, 97, 106);
height:3px;}
QSlider::sub-page:Horizontal {
background-color:rgb(40, 115, 216);
height:3px;}
QSlider::groove:Horizontal{
background:transparent;
height:3px;}
QSlider::handle:Horizontal{
height: 12px;
width:12px;  
border-image: url(:/res/images/slider_handle.png);
margin: -4 -2px;  }</string>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="singleStep">
     <number>3</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>30</y>
      <width>15</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>15</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/images/play.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string>00:00:00/00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>27</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>27</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>27</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/images/openfile.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiniPlayerScreenWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">miniplayerscreenwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="miniplayer.qrc"/>
 </resources>
 <connections/>
</ui>
